### PR TITLE
Add support for dropping views in MSSQL server when running migrate/fresh

### PR DESCRIPTION
**Pull request by ambrozt**
Add support for dropping views in MSSQL server when running migrate/fresh

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #20256

### CODE REVIEW

File: framework/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Yii Framework 2 Change Log
 - Bug #20232: Fix regression introduced in `GHSA-cjcc-p67m-7qxm` while attaching behavior defined by `__class` array key (erickskrauch)
 - Bug #20231: Fix regression introduced in #20167 in `yii\validators\FileValidator` (bizley)
 - Enh #20247: Support for variadic console controller action methods (brandonkelly)
+- Enh #20256: Added missing MSSQL condition in `yii\console\controllers\MigrateController` isViewRelated function (ambrozt)
```
Comment (samdark) [inline framework/CHANGELOG.md:10]:
```suggestion
- Bug #20256: Add support for dropping views in MSSQL server when running migrate/fresh (ambrozt)
```

**Review by samdark — APPROVED** [review]
Good one. Thanks. Possible to add test for it?

**Review by mtangoo — APPROVED** [review]
Looks good to me. If no one else have objection, I will merge it!

**Review by Arhell — APPROVED** [review]

**Comment by mtangoo** [comment]
@ambrozt Thanks for your time and efforts

**Comment by ambrozt** [comment]
> Good one. Thanks. Possible to add test for it?

I think the function testRefreshMigration in yiiunit\framework\console\controllers\MigrateControllerTest should already cover this scenario, although I am not sure why the test is not being run on Microsoft SQL databases

**Pull request merged**